fipy/tools/numerix.py

```diff
@@ -222,10 +226,11 @@ def rank(a):
 
     .. note::
 
-       The rank of a `MeshVariable` is for any single element. E.g., A
-       `CellVariable` containing scalars at each cell, and defined on a 9
-       element `Grid1D`, has rank 0. If it is defined on a 3x3 `Grid2D`, it is
-       still rank 0.
+       The rank of a :class:`~fipy.variables.meshVariable.MeshVariable` is
```

Comment:
Why am I seeing [this](https://pages.nist.gov/fipy/en/modernize_sphinx/generated/fipy.tools.numerix.html#fipy.tools.numerix.rank) on pages.nist.gov?

Reply:
Sorry, all good. I'm seeing the correct version